applications/standalone/src/mock/resources/sampleLumyWorkflow.yml

```diff
@@ -0,0 +1,22 @@
+processing:
+  workflow:
+    name: test
+ui:
+  pages:
+    - id: dataSelection
+      component:
+        id: dataSelection
+      meta:
+        label: Data Selection
+    - id: dataMapping
+      component:
+        id: networkAnalysisDataMapping
+      meta:
+        label: Data mapping
+    - id: datavis
+      component:
+        id: networkAnalysisDataVis
+      meta:
+        label: Data visusalisation
```

Comment:
Thanks for adding step/page labels. This is great! There's a minor typo here, though:
`visu**s**alisation`

Reply:
thanks!